doc/toolchain.rst

```diff
@@ -124,9 +124,16 @@ if a unit defines ``%port = 80/tcp``, you can use ``spicy-driver -p
 tagged with an ``&originator`` or ``&responder`` attribute,
 respectively, will be considered. If a unit defines ``%mime-type =
 application/test``, you can select it through ``spicy-driver -p
-application/test``. (Note that there must be exactly one unit with a
-matching property for this all to work, otherwise you'll get an error
-message.)
+application/test``. (Note that there must be exactly one unit with a matching
+property for this all to work, otherwise you'll get an error message.)
```

Comment:
I would (personally) prefer the parentheses be within the previous sentence:

> you can select it through ``spicy-driver -p
application/test`` (note that there must be exactly one unit with a matching property for this all to work, otherwise you'll get an error message).

Reply:
Actually I think it's better separate because the note applies to much of the paragraph, not just the one sentence. That said, I've been going back and forth over this already because with the new extended paragraph, the note feel a bit confusing. I'll just go ahead and remove it, I don't think it's important anyways, it was just for completeness.